standard/league_icon.lua

```diff
@@ -57,14 +57,15 @@ function LeagueIcon._make(icon, iconDark, link, name, size)
 end
 
 --retrieve icon and iconDark from LeagueIconSmall templates
-function LeagueIcon.getIconFromTemplate(icon, iconDark, series, abbreviation, date)
-	local leagueIconSmallTemplate = LeagueIcon._getTemplate(series, abbreviation, date)
+function LeagueIcon.getIconFromTemplate(icon, iconDark, series, abbreviation, date, leagueIconSmallTemplate)
```

Comment:
It's not clear to me what the purpose of this param is. The name is pretty vague and there's no docs either

Reply:
added a comment

Reply:
Okay clearer to me now. Would it not be easier to extract the parsing part of this function (e.g. the split) and make that a new function, which we can then call instead? It seems to me we're sort of ruining the cleanness of this function with this change

Reply:
moved the getTemplate out of the function

and adjusted the functions a bit to be better usable from wiki code